Description: The library bzlib is included in the source and
  built as a convenience library. This patch prevents this from being
  built and links berkeley-abc with debian's bzlib library instead.
Forwarded: doesn't make sense upstream
Author: Ruben Undheim <ruben.undheim@gmail.com>
Index: yosys-abc/Makefile
===================================================================
--- yosys-abc.orig/Makefile
+++ yosys-abc/Makefile
@@ -20,7 +20,7 @@ MODULES := \
 	src/map/mapper src/map/mio src/map/super src/map/if \
 	src/map/amap src/map/cov src/map/scl src/map/mpm \
 	src/misc/extra src/misc/mvc src/misc/st src/misc/util src/misc/nm \
-	src/misc/vec src/misc/hash src/misc/tim src/misc/bzlib src/misc/zlib \
+	src/misc/vec src/misc/hash src/misc/tim src/misc/zlib \
 	src/misc/mem src/misc/bar src/misc/bbl \
 	src/opt/cut src/opt/fxu src/opt/rwr src/opt/mfs src/opt/sim \
 	src/opt/ret src/opt/res src/opt/lpk src/opt/nwk src/opt/rwt \
@@ -67,7 +67,7 @@ endif
 endif
 
 # LIBS := -ldl -lrt
-LIBS := -ldl
+LIBS := -ldl -lbz2
 ifneq ($(findstring Darwin, $(shell uname)), Darwin)
    LIBS += -lrt
 endif
Index: yosys-abc/src/base/io/ioReadAiger.c
===================================================================
--- yosys-abc.orig/src/base/io/ioReadAiger.c
+++ yosys-abc/src/base/io/ioReadAiger.c
@@ -26,7 +26,7 @@
 #include <string.h>
 #include <assert.h>
 
-#include "misc/bzlib/bzlib.h"
+#include <bzlib.h>
 #include "misc/zlib/zlib.h"
 #include "ioAbc.h"
 
Index: yosys-abc/src/base/io/ioWriteAiger.c
===================================================================
--- yosys-abc.orig/src/base/io/ioWriteAiger.c
+++ yosys-abc/src/base/io/ioWriteAiger.c
@@ -26,7 +26,7 @@
 #include <string.h>
 #include <assert.h>
 
-#include "misc/bzlib/bzlib.h"
+#include <bzlib.h>
 #include "misc/zlib/zlib.h"
 #include "ioAbc.h"
 
